components/BannerManager/BannerManager.tsx

```diff
@@ -14,18 +14,23 @@ import { snapshotEndpoint } from 'constants/snapshot';
 
 const BannerManager: FC = () => {
 	const {
+		issuance,
 		useGetLiquidationDataQuery,
 		useGetDebtDataQuery,
 		useHasVotedForElectionsQuery,
-		useFeeClaimHistoryQuery,
 	} = useSynthetixQueries();
 
 	const walletAddress = useRecoilValue(walletAddressState);
 
 	const liquidationData = useGetLiquidationDataQuery(walletAddress);
 	const debtData = useGetDebtDataQuery(walletAddress);
 	const hasVotedForElectionsQuery = useHasVotedForElectionsQuery(snapshotEndpoint, walletAddress);
-	const feeClaimHistory = useFeeClaimHistoryQuery(walletAddress);
+
+	const feeClaims = issuance.useGetFeesClaimeds(
```

Comment:
I'm not a native speaker, but what is claimeds?

Reply:
I thing the code generating these hooks takes the name and adds `s` for the hook that return an array.
So the name for this is getFeesClaimed, then `use` since its a hook and `s` since it's return an array